### PR TITLE
feat:add input component

### PR DESCRIPTION
**Pull request by rahulmistry751**
PR: Input component
- Normal input
- Input with validation
- Textarea
- Slider
input

### CODE REVIEW

File: components/css/input/input.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
+.input-container{
+    width:20%;
+    min-width:300px;
```
Comment (Nishant-96) [inline components/css/input/input.css]: Suggestion Try to use relative units.

File: components/css/input/input.css [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    border:none;
+    
+  }
+.input:hover,.textarea:hover{
+    border:1px solid #D1D5DB;
+    transition:.5s;
+  }
+.input:focus, .textarea:focus{
+    border:none;
+    box-shadow:0px 0px 12px #D1D5DB;
```
Comment (Nishant-96) [inline components/css/input/input.css]: Suggestion  if you are using same color at many places you can use variable instead of repeating yourself.

**Review by Nishant-96 — APPROVED** [review]

File: components/css/input/input.css [hunk 3]
```diff
@@ -0,0 +1,38 @@
+.input-container{
+    width:20%;
+    min-width:300px;
+  }
+.input-container *{
+    margin:12px;
```
Comment (znjs) [inline components/css/input/input.css]: Try using relative units

**Review by znjs — APPROVED** [review]

**Review by ayush65 — APPROVED** [review]

File: docs/pages/input.html [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+                </p>
+                <hr>
+                <h3 class="content-title h3 fw-700">Input</h3>
+                <div class="example-container fx-c">
+                   <!-- Input -->
+                    <div class="input-container">
+                        <label for="">
+                            Username
+                        </label>
+                        <input type="text" class="input">
```
Comment (narharikale) [inline docs/pages/input.html:66]: is there any specific reason to not pass any value in the` for=""` attribute ??

To associate the `<label>` with an `<input>` element, you need to give the `<input>` an id attribute. The `<label>` then needs a for attribute whose value is the same as the input's id:
-e.g `<label for="input1">Username</label>
                                                                         <input type="text" class="input">`

Comment (rahulmistry751) [reply docs/pages/input.html:66]: yeah. I kept it blank deliberately. I thought user could add it's own id name depending on the application.

File: components/css/input/input.css (hunk 1 above)
Comment (rahulmistry751) [reply components/css/input/input.css]: yes. thanks

**Review by narharikale — APPROVED** [review]

**Pull request merged**